src/app/politician-page/politician-page.component.html

```diff
@@ -1,28 +1,44 @@
 <div class="container">
-
   <!--Politician profile card-->
-  <div class="row">
+  <div class="row equal">
 
     <!--Politician image-->
-    <div class="col-2">
-      <img [src]="Politician.bild_url_192">
+    <div class="col-sm-4">
+      <img [src]="Politician.bild_url_192" class="img-fluid" alt="">
     </div>
 
     <!--Politician general info-->
-    <div class="col-5">
-      <h1>{{Politician.sorteringsnamn}}</h1>
-      <h3>{{Politician.status}}</h3>
-      <h3>{{Politician.valkrets}}</h3>
-      <h3>{{Politician.parti}}</h3>
+    <div class="col-sm-8">
+
+      <div class="row">
+        <div class="col">
+          <h1>{{Politician.tilltalsnamn}} {{Politician.efternamn}}</h1>
+          <h3>{{Politician.status}}</h3>
+          <h3>{{Politician.parti}}</h3>
+        </div>
+      </div>
+
+      <div class="row politician-info">
+        <!--Politician contact info-->
+        <div class="col">
+          <ul>
+            <li><strong>Email</strong><span>{{Politician.tilltalsnamn}}.{{Politician.efternamn}}@riksdagen.se</span></li>
```

Comment:
Might be an unreliable way of linking to their email. That said, I did not find the email in the politician data.